moveit_planners/ompl/ompl_interface/include/moveit/ompl_interface/detail/constrained_valid_state_sampler.h

```diff
@@ -57,9 +57,9 @@ class ValidConstrainedSampler : public ompl::base::ValidStateSampler
       const ModelBasedPlanningContext* pc, const kinematic_constraints::KinematicConstraintSetPtr& ks,
       const constraint_samplers::ConstraintSamplerPtr& cs = constraint_samplers::ConstraintSamplerPtr());
 
-  virtual bool sample(ompl::base::State* state);
+  bool sample(ompl::base::State* state) override;
   virtual bool project(ompl::base::State* state);
-  virtual bool sampleNear(ompl::base::State* state, const ompl::base::State* near, const double distance);
+  bool sampleNear(ompl::base::State* state, const ompl::base::State* near, const double distance) override;
```

Comment:
i assume you ran clang-tidy or clang-modernize on this package? could you do that in a preliminary PR first so that reviewing this one is much easier?